### PR TITLE
Analytics, login-or-register, signup

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/signup/breadcrumbs.tsx [hunk 1]
```diff
@@ -11,19 +11,29 @@ const Container = styled.div`
...
+  inOpacity?: boolean;
+  inActive: boolean;
 }
 
 const Phase = styled.div`
   display: flex;
-  opacity: ${(props: PhaseProps) => (props.inActive ? 0.5 : 1)};
+  opacity: ${(props: PhaseProps) => (props.inOpacity ? 0.5 : 1)};
+  @media only screen and (max-width: ${MOBILE_WIDTH}px) {
+    display: ${(props: PhaseProps) => (props.inActive ? "block" : "none")};
```
Comment (EdmundMai) [inline src/components/signup/breadcrumbs.tsx]: It makes more sense for this to be named `isActive`.

`isActive ? "block" : "none";`

File: src/components/signup/breadcrumbs.tsx [hunk 2]
```diff
@@ -11,19 +11,29 @@ const Container = styled.div`
...
 interface PhaseProps {
-  inActive?: boolean;
+  inOpacity?: boolean;
+  inActive: boolean;
 }
 
 const Phase = styled.div`
   display: flex;
-  opacity: ${(props: PhaseProps) => (props.inActive ? 0.5 : 1)};
+  opacity: ${(props: PhaseProps) => (props.inOpacity ? 0.5 : 1)};
```
Comment (EdmundMai) [inline src/components/signup/breadcrumbs.tsx]: Lets rename `inOpacity` to `isHighlighted`

`isHighlighted ? 1 : 0.5`

File: src/components/signup/breadcrumbs.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/signup/breadcrumbs.tsx]: fixed

File: src/components/signup/breadcrumbs.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/signup/breadcrumbs.tsx]: fixed

File: src/components/login-or-register/login-or-register-slide.tsx [hunk 3]
```diff
@@ -166,30 +168,32 @@ const Content = styled.div`
...
-        </Container>
-      </Storage>
-    </StorageContainer>
+      <StorageContainer>
+        <Storage>
+          <Container>
+            <Icon src={ICON_LOGO} />
+          </Container>
+          <Container>
+            <Title>Sing in Opacity</Title>
```
Comment (rfornea) [inline src/components/login-or-register/login-or-register-slide.tsx]: Sing?

**Pull request merged**